### PR TITLE
Add Integration Test

### PR DESCRIPTION
**Pull request by kemurayama**
Added Integration test for space-agon

Closes (#10 )

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -62,6 +62,9 @@ help:
 	@echo "Setup a Skaffold file for debugging !!RUN AFTER CREATING YOUR CLUSTER!!"
 	@echo "    make skaffold-setup"
 	@echo ""
+	@echo "Run integration test. "
```
Comment (govargo) [inline Makefile:65]: remove last blank

File: README.md [hunk 2]
```diff
@@ -158,6 +158,24 @@ skaffold dev
...
 Modifying applications during `skaffold dev` triggers Build and Deploy automatically. For more commands and details, visit [`skaffold`](https://skaffold.dev/). 
 
+### Test the Cluster
+
+When you would like to test the application, follow the steps below.
+
+1. [Install `skaffold`](https://skaffold.dev/docs/install/) if you haven't. 
+1. [Create a space-agon k8s cluster.](#create-the-resources-and-install-gaming-oss)
+1. Run `make skaffold-setup` on the project root to make a `skaffold.yaml`
+1. Run below commands for E2E test.
```
Comment (govargo) [inline README.md:168]: E2E test → integration test

File: Makefile (hunk 1 above)
Comment (kemurayama) [reply Makefile]: Thanks, I also deleted the period.

File: README.md (hunk 2 above)
Comment (kemurayama) [reply README.md]: Changed to Integration test.

**Comment by govargo** [comment]
LGTM

**Review by govargo — APPROVED** [review]

**Pull request merged**